include/openssl/err.h

```diff
@@ -339,6 +339,16 @@ OPENSSL_EXPORT int ERR_get_next_error_library(void);
 #define ERR_LIB_TRUST_TOKEN     32
 #define ERR_LIB_USER            33
 #define ERR_NUM_LIBS            34
+#define ERR_LIB_PKCS12          35
+#define ERR_LIB_DSO             36
+#define ERR_LIB_OSSL_STORE      37
+#define ERR_LIB_FIPS            38
+#define ERR_LIB_CMS             39
+#define ERR_LIB_TS              40
+#define ERR_LIB_CT              41
+#define ERR_LIB_ASYNC           42
+#define ERR_LIB_KDF             43
+#define ERR_LIB_SM2             44
```

Comment:
Hmmmmmmm, does NodeJS not handle `OPENSSL_NO_SM2` correctly?

Reply:
Not sure but they define mappings to these specific ERR_LIB_* codes, so we need to define them as well.

Reply:
Where do you see that? A naive search only finds the OpenSSL dependency code and no real usage https://github.com/search?q=repo%3Anodejs%2Fnode%20ERR_LIB_SM2&type=code

Reply:
They create a mapping to those err codes here:
https://github.com/nodejs/node/blob/1abff073921bcb0631602032aef0135bccfaee0d/src/crypto/crypto_util.cc#L501